### PR TITLE
Ryan italy

### PR DESCRIPTION
**Pull request by ryanmadamba**
Pull Request for some updated styling for Beta Release

### CODE REVIEW

File: src/Components/CourseScreen.css [hunk 1]
```diff
@@ -72,26 +72,24 @@
   font-size: 15px;
   padding-right:5px;
 }
-
-/* h3 {
```
Comment (Bart-Ilg) [inline src/Components/CourseScreen.css:76]: good to clear commented code that isnt sneeded

File: src/Components/Courses.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ function Course() {
 
   return (
     <div> 
-      <h1>My Courses</h1>
+      <h6>My Courses</h6>
```
Comment (Bart-Ilg) [inline src/Components/Courses.js:81]: good format change

File: src/Components/MissionsScreen.css [hunk 3]
```diff
@@ -60,4 +60,20 @@
     box-shadow: 0 5px 15px rgba(0,0,0,0.3);
     background-color: navy;
     cursor: pointer;
+}
+
+h1 {
+    padding-top: 75px;
+}
+
+.previewCard {
```
Comment (Bart-Ilg) [inline src/Components/MissionsScreen.css:69]: clean formatting

**Review by tddumont — APPROVED** [review]
Looks great!

**Review by Bart-Ilg — APPROVED** [review]
Looks good, mostly styling changes so not too logic heavy in examination, but looks good in running product.

**Pull request merged**